index.bs

```diff
@@ -320,18 +315,18 @@ The <dfn method for=IdleDetector>start(|options|)</dfn> method steps are:
     <wpt>
       interceptor.https.html
     </wpt>
-1.  [=Queue a global task=] on the [=relevant global object=] of [=this=] using
-    the [=idle detection task source=] to perform the following steps, but
-    [=abort when=] |this|.{{IdleDetector/[[state]]}} becomes `"stopped"`.
-    1.  Let |permissionState| be the [=permission state=] the [=powerful
-        feature=] named `"idle-detection"`.
+1. [=In parallel=]:
+    1.  Let |permissionState| be the [=permission state=] of `"idle-detection"`.
     1.  If |permissionState| is `"denied"`, [=reject=] |result| with a
```

Comment:
We need to queue a task here to do the rejection.

Reply:
Right, yes. Um, on the "permissions task source" or the "idle detection task source"? 

I'm thinking it would be on the "permissions task source", as it's in that part of the code. Will send that and we change it quickly if wrong.

Reply:
Actually, never mind... it's on the idle detection task source, as it changes the state machine.